.cspell.json

```diff
@@ -141,6 +141,7 @@
         "kadda",
         "keldon",
         "keldon's",
+	"keyable",
```

Comment:
looks like you manually inserted this instead of using the cspell took, because the whitespace is wrong

Reply:
I didn't know cspel can edit this file. Fixed.